frontend/src/metabase/admin/settings/components/SettingsGoogleForm.jsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/prop-types */
```

Comment:
Let's please not introduce new prop type errors (even if we ignore them). This should be handled before the merge.

Reply:
Ah, sorry, I copied this bit from `SettingsSAMLForm`. I'll change it.